vendor/modules.txt

```diff
@@ -73,6 +73,8 @@ github.com/google/go-cmp/cmp
 github.com/google/go-cmp/cmp/internal/diff
 github.com/google/go-cmp/cmp/internal/function
 github.com/google/go-cmp/cmp/internal/value
+# github.com/google/go-querystring v1.0.0
```

Comment:
This and anything below actually came as a result of `go mod vendor` and wasn't intentional. Not really a go developer by myself so I left with the assumption that `go mod` is smarter than me.

Reply:
Yeah I'd leave it in there if go mod wants to add it.

Reply:
I removed the vendoring. So with a rebase this should solve itself aswell.